bpaotu/bpaotu/settings.py

```diff
@@ -334,14 +334,13 @@
 
     'default': {
         "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": env.getlist("cache", ["redis://cache:6379/1"]),
+        "LOCATION": env.getlist("cache", ["redis://otucache:6379/1"]), # NOTE: Passing in via Docker yml does not work.
```

Comment:
could you revert this out from this commit and fix the underlying issue? don't want to put workarounds into the master branch

Reply:
Need your help for this one. :)